qutebrowser/config/configfiles.py

```diff
@@ -198,6 +199,13 @@ def read_config_py(filename=None):
         if not os.path.exists(filename):
             return api
 
+    # Add config directory to python path, so config.py can import other files
+    # in logical places
+    old_state = _pre_config_save()
+    config_dir = os.path.dirname(filename)
+    if config_dir not in sys.path:
+        sys.path = [config_dir] + sys.path
+
```

Comment:
I think it'd be better to do this as late as possible, i.e. right before the `exec`.